wdl-core/src/concern/lint/group.rs

```diff
@@ -9,6 +9,9 @@ pub enum Group {
     /// Rules associated with having a complete document.
     Completeness,
 
+    /// Rules associated with the names of WDL elements.
+    Naming,
```

Comment:
I see now that you were going off of proposed groups in [this discussion](https://github.com/stjude-rust-labs/wdl/discussions/11). Given that we've thought this through at a higher level, I'm okay with keeping this group now.

Reply:
> these should just be `Style`—that term probably is a superset of what `Naming` would be.

I think you're correct that `Naming` will be a subset of `Style`. I think that's actually a great argument for the idea I raised in #12 about using "tags" instead of a singular "group". The broadness of `Style` caters to a different use case than the specificity of `Naming`. We should support both, although that is of course a larger change OOS for this PR. In the meantime, I don't feel strongly for one over the other.

Reply:
Yeah, fair enough. We can go with this for now and then change to tags in the future.